lib/YaST/Bootloader/BootloaderOptionsTab.pm

```diff
@@ -0,0 +1,34 @@
+# SUSE's openQA tests
+#
+# Copyright 2021 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: The class introduces all accessing methods for Bootloader Options tab
+# in Boot Loader Settings.
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+package YaST::Bootloader::BootloaderOptionsTab;
+use parent 'Installation::Navigation::NavigationBase';
+use strict;
+use warnings;
+use testapi;
+
+sub init {
+    my $self = shift;
+    $self->SUPER::init();
```

Comment:
this class doesn't need to container NavigationBase or any libyui-rest-api,
to check that you are in the right screen, you just need existing method:
`get_boot_code_options_page`. I guess in the current design we would need another controller which would include `BootCodeOptionsPage` in similar way that `BootloaderSettingsController` does plus the needling code below.

Reply:
I need this code: $self->{cmb_bootloader} = $self->{app}->combobox({id => "\"Bootloader::LoaderTypeWidget\""});
To make sure the page: https://openqa.suse.de/tests/10917817#step/disable_boot_menu_timeout_navigating_tabs/1 is shown
After this, I can use right key to get 'bootloader options' page